src/convert.ts

```diff
@@ -148,6 +153,12 @@ function mh2bytes (hash: string) {
   return uint8ArrayConcat([size, mh], size.length + mh.length)
 }
 
+function mb2bytes(mbstr: string) {
+  let mb = MB.decode(mbstr)
+  const size = Uint8Array.from(varint.encode(mb.length))
+  return uint8ArrayConcat([size, mb], size.length + mb.length)
```

Comment:
is there any verification required that the decoded multibase contains a valid multihash?

Reply:
I'm not sure there's an easy way to verify that. The existing multihash function does one validation I did not replicate here, 

https://github.com/multiformats/js-multiaddr/blob/master/src/convert.ts#L158-L160

That it reports a size that is correct. I could (maybe should) do that. But to go beyond that I think we'd have to be aware of all the supported encryption schemes, correct?

Reply:
@achingbrain  - thoughts on this?